src/pybind/mgr/rook/module.py

```diff
@@ -134,6 +134,20 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class RookEnv(object):
+    def __init__(self):
+        pod_namespace = os.environ.get('POD_NAMESPACE', 'rook-ceph')
+        self.cluster_name = os.environ.get('ROOK_CLUSTER_NAME', pod_namespace)
+        self.cluster_ns = os.environ.get('ROOK_CLUSTER_NS', self.cluster_name)
+
+        self.system_ns = os.environ.get('ROOK_SYSTEM_NS', "rook-ceph-system")
+        self.api_version = os.environ.get('ROOK_API_VERSION', 'v1')
```

Comment:
Can we change this to `ROOK_CEPH_CRD_VERSION`

Reply:
This has been renamed to `ROOK_CEPH_CLUSTER_CRD_VERSION` in https://github.com/rook/rook/pull/2903

Reply:
Nah, I'm going to wait for another day or two, till I rename them again.